test/gemm/gemm_kernel_base_dispatch_impl.hpp

```diff
@@ -81,7 +81,7 @@ namespace rocwmma
         waveSize, SWITCH_BODY_TBLOCK_Y, HipDevice::Wave32, HipDevice::Wave64, ARCH_ID)
 
 #define DISPATCH_GUARD_BODY                           \
-    ROCWMMA_SWITCH_BODY10_ARG1(deviceArch,            \
+    ROCWMMA_SWITCH_BODY11_ARG1(deviceArch,            \
```

Comment:
Eventually, it'd be real nice to have a list macro to consume with all of the supported targets, instead of having to go to each usage site and update them one by one. Maybe something to keep in mind for the future :)